autoscale-email-alert-dispatcher/pom.xml

```diff
@@ -33,17 +33,23 @@
         <dependency>
             <groupId>com.github.autoscaler</groupId>
             <artifactId>autoscale-api</artifactId>
-            <version>1.5.0-SNAPSHOT</version>
         </dependency>
         <dependency>
             <groupId>javax.mail</groupId>
             <artifactId>mail</artifactId>
-            <version>1.5.0-b01</version>
         </dependency>
         <dependency>
             <groupId>javax.validation</groupId>
             <artifactId>validation-api</artifactId>
             <scope>provided</scope>
```

Comment:
Should this scope be provided?

Reply:
Provided scope removed as it is not needed.

Reply:
I think Dermot mentioned that this dependency is ok to have provided scope as you do not want this at runtime.